### PR TITLE
New tokens/alpha charlie

### PR DESCRIPTION
**Pull request by therealMikkoK**

### CODE REVIEW

**Comment by therealMikkoK** [comment]
@freeatnet some links were unexpected, and didn't work. We have to look how to archive the PDFs in the future.

File: tokens/0x4f3afec4e5a3f2a6a1a411def7d7dfe50ee057bf.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+decimals: 9
+description: >-
+  We create a world where the finest gold bars are made divisible and transferable on the blockchain.
+  The digital gold standard is here.
+links:
+- Chat: https://discord.gg/CCDBJJC
+- CoinGecko: https://www.coingecko.com/en/coins/digix-gold
+- CoinMarketCap: https://coinmarketcap.com/currencies/digix-gold-token/
+- Reddit: https://www.reddit.com/r/digix/
+- Source Code: https://gist.github.com/mrenoon/2582fba7b4d457d80f7d37520aabbc08
```
Comment (freeatnet) [inline tokens/0x4f3afec4e5a3f2a6a1a411def7d7dfe50ee057bf.yaml]: > Non-critical issue: can 'Source Code: https://gist.github.com/mrenoon/2582fba7b4d457d80f7d37520aabbc08' be better described by one of well-known types: frozenset({'Bitcointalk', 'CoinMarketCap', 'CoinGecko', 'Slack', 'Website', 'Github', 'Whitepaper', 'Facebook', 'WeChat', 'Email', 'Instagram', 'Reddit', 'Telegram', 'Twitter', 'Blog', 'Chat', 'YouTube', 'Discord', 'Linkedin'})?

File: tokens/0xc56b13ebbcffa67cfb7979b900b736b3fb480d78.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+decimals: 8
+description: >-
+  Decentralized social network
+  You take back power with every social connection
+links:
+- Website: https://sphere.social/hello/
+- Google Play: https://play.google.com/store/apps/details?id=com.sphere.social
+- Whitepaper: https://sphere.social/wp-content/uploads/2017/12/Sphere_Whitepaper_v1.7.4.pdf
+name: Social Activity Token
+symbol: SATO
```
Comment (freeatnet) [inline tokens/0xc56b13ebbcffa67cfb7979b900b736b3fb480d78.yaml]: > 4) Non-critical issue: expected symbol to be SAT, but got SATO
        in test_symbol_equals_erc20_symbol: name should reflect the symbol defined by the contract

@therealMikkoK Looks like we replaced "SAT" with "SATO" in https://github.com/forkdelta/tokenbase/commit/70cd2f48d0976f25c674db9708330a2bb4e48781. Let's use `symbol: SAT` and put the custom "SATO" in `__FORKDELTA_CUSTOM_SYMBOL`: `__FORKDELTA_CUSTOM_SYMBOL: SATO`.

File: tokens/0xc5bbae50781be1669306b9e001eff57a2957b09d.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+---
+addr: '0xc5bbae50781be1669306b9e001eff57a2957b09d'
+decimals: 5
+description: >-
+  Create meaninfull and long lasting connections within communities through the power of blockchain gifting.
+links:
+- Blog: https://medium.com/@gifto
+- Chat 2: https://open.kakao.com/o/ggLQMiC
```
Comment (freeatnet) [inline tokens/0xc5bbae50781be1669306b9e001eff57a2957b09d.yaml]: > Non-critical issue: can 'Chat 2: https://open.kakao.com/o/ggLQMiC' be better described by one of well-known types: frozenset({'Bitcointalk', 'CoinMarketCap', 'CoinGecko', 'Slack', 'Website', 'Github', 'Whitepaper', 'Facebook', 'WeChat', 'Email', 'Instagram', 'Reddit', 'Telegram', 'Twitter', 'Blog', 'Chat', 'YouTube', 'Discord', 'Linkedin'})?

We can probably use known type "Chat" here.

File: tokens/0xc5bbae50781be1669306b9e001eff57a2957b09d.yaml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+---
+addr: '0xc5bbae50781be1669306b9e001eff57a2957b09d'
+decimals: 5
+description: >-
+  Create meaninfull and long lasting connections within communities through the power of blockchain gifting.
```
Comment (freeatnet) [inline tokens/0xc5bbae50781be1669306b9e001eff57a2957b09d.yaml]: > meaninfull

Typo.

File: tokens/0xc56b13ebbcffa67cfb7979b900b736b3fb480d78.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+---
+__FORKDELTA_CUSTOM_SYMBOL: AVOID CONFLICT WITH 0xd9a0658b7cc9ec0c57e8b20c0920d08f17e747be
+addr: '0xc56b13ebbcffa67cfb7979b900b736b3fb480d78'
+decimals: 8
+description: >-
+  Decentralized social network
+  You take back power with every social connection
```
Comment (freeatnet) [inline tokens/0xc56b13ebbcffa67cfb7979b900b736b3fb480d78.yaml]: Can we find a better description here?

**Review by freeatnet — COMMENTED** [review]
Needs a little more attention.

File: tokens/0x80238d9451f86dcf12bff21af9e8deadcf915bff.yaml [hunk 6]
```diff
@@ -0,0 +1,14 @@
+---
+addr: '0x80238d9451f86dcf12bff21af9e8deadcf915bff'
+decimals: 18
+description: >-
+  Being an intermediary between various entities involved in the ticket reselling process,
+  the platform serves as a guarantor of transaction security, tickets’ validity, timely payment and delivery.
+links:
+- Website: https://et4.io
```
Comment (freeatnet) [inline tokens/0x80238d9451f86dcf12bff21af9e8deadcf915bff.yaml]: This [is failing](https://travis-ci.org/forkdelta/tokenbase/builds/437782448#L575) with an `SSLError`. This is a test problem, I'll see if I can do anything about it.

File: tokens/0xc56b13ebbcffa67cfb7979b900b736b3fb480d78.yaml (hunk 5 above)
Comment (therealMikkoK) [reply tokens/0xc56b13ebbcffa67cfb7979b900b736b3fb480d78.yaml]: I tried to find some, but as they've launched it seems they let the platform talk for itself.
My wish as well

File: tokens/0xc56b13ebbcffa67cfb7979b900b736b3fb480d78.yaml (hunk 2 above)
Comment (therealMikkoK) [reply tokens/0xc56b13ebbcffa67cfb7979b900b736b3fb480d78.yaml]: Need assistance on this @freeatnet

**Pull request closed without merging**